std_msgs/Header header

float64 mask

float64 rang0
float64 rang1
float64 rang2
float64 rang3

float64 nRangs
float64 resQ
float64 debug
